### DISCUSSION TITLE
Feature request: relay priority

### DISCUSSION THREAD

**Discussion by thegostep** [flashbots/pm#29] [searchers]
The relay currently treats all incoming bundles equally and determines its validity before forwarding to mev-geth miners in order to mitigate DOS.

It would be possible to accelerate the processing for searchers with a good history of submissions or some economic skin in the game.

The details of the mechanism remain to be defined.

Issue: https://github.com/flashbots/mev-relay-js/issues/30

**Comment by Napcalx** [comment]
That's good
